Riot/Modules/VoiceBroadcast/VoiceBroadcastSDK/VoiceBroadcastService.swift

```diff
@@ -156,6 +160,10 @@ public class VoiceBroadcastService: NSObject {
             
             voiceBroadcastInfo.state = state.rawValue
             
+            if let lastChunkSequence = lastChunkSequence {
+                voiceBroadcastInfo.lastChunkSequence = lastChunkSequence
+            }
+            
```

Comment:
`lastChunkSequence` is not initialized correctly here, we should add :
`else voiceBroadcastInfo.lastChunkSequence = 0`

Why don't we use the (initWithDeviceId...)[https://github.com/vector-im/element-ios/pull/7136/files#diff-3984ce62497db3e36cab4bcb87c1bb1d97d140510c5acba01b76c0b907d189caR22] method to init `voiceBroadcastInfo` instance?

Reply:
If I change this part like this, `lastChunkSequence` will be sent with every `VoiceBroadcastInfo` (start, pause, stop and resume).

Reply:
Updated in last commit.